release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1275,25 +1280,40 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
+|{sno-caps} cluster expansion with worker nodes
```

Comment:
For this and the following 2 features, I have a question. If all 3 releases are 'General Availability', what is the purpose to list them under 'Technology Preview features' section?

Reply:
@qiliRedHat  Thank you for asking! We still support the last four releases and we have customers on those releases, therefore we must show the history of the feature in past and current release notes.  All sections have the name "Technology Preview features" and list the features even after they have gone General Availability.  We do this for legal requirements and for tech support requirements.